include/internal/catch_tostring.h

```diff
@@ -210,6 +210,13 @@ namespace Catch {
         }
     };
 
+#if defined(CATCH_CONFIG_CPP17_BYTE)
+#include <cstddef>
```

Comment:
You are including a standard library header inside a namespace... this is not a good idea.

Because this file already uses `<cstddef>` by default, it is possible to just remove this. Alternatively, it should be included in a manner similar to `<string_view>` around line 20.

Reply:
Doh! I'll fix that...